[XPU]change xpu ci model

plusNew001 opened the pull request:
更改XPU CI 使用的模型

Commit:
change xpu ci model

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/FastDeploy?pullRequest=4117) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/FastDeploy?pullRequest=4117) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: plusNew001<br/>:x: root<br/><hr/>**root** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/FastDeploy?pullRequest=4117) it.</sub>

plusNew001 committed:
change xpu ci model

plusNew001 committed:
change xpu ci model

plusNew001 committed:
change xpu ci model

plusNew001 committed:
Merge branch 'PaddlePaddle:develop' into xpu_ci

plusNew001 committed:
Update model path and XPU settings in run_ci_xpu.sh

plusNew001 committed:
Increase health check timeout to 10 minutes

Increased the timeout duration for health checks from 5 minutes to 10 minutes in two places.

plusNew001 committed:
Implement test for OpenAI chat completion

Add a test function for the OpenAI client chat response.

plusNew001 committed:
Change script to use pytest for running tests

plusNew001 committed:
Update health check timeout to 15 minutes

Increase the timeout for health checks from 10 minutes to 15 minutes.

plusNew001 committed:
Add pytest installation to CI script

plusNew001 committed:
Modify base response in test_45t function

Updated the base response message for the test.

plusNew001 committed:
Add V0 and V1 mode test echo statements
